_source/handbook/02_drive.md

```diff
@@ -304,6 +304,35 @@ If the form submission is a GET request, you may render the directly rendered re
 
 Servers may also respond to form submissions with a [Turbo Streams](streams) message by sending the header `Content-Type: text/vnd.turbo-stream.html` followed by one or more `<turbo-stream>` elements in the response body. This lets you update multiple parts of the page without navigating.
 
+## Instaclick
```

Comment:
Name of the library is InstantClick, though not sure if we're going with a different one on the docs, or if we should call it "Prefetching" instead.

"InstantClick" might help with SEO for users searching for it.